lib/alchemy/page_layout.rb

```diff
@@ -44,7 +44,7 @@ def get_all_by_attributes(attributes)
         if attributes.class.name == 'Hash'
           layouts = []
           attributes.stringify_keys.each do |key, value|
-            result = all.select { |a| a[key].to_s.downcase == value.to_s.downcase if a.has_key?(key) }
+            result = all.select { |a| a[key].to_s.downcase == value.to_s.downcase if a.key?(key) }
```

Comment:
Performance/Casecmp: Use `casecmp` instead of `downcase ==`. (https://github.com/JuanitoFatas/fast-ruby#stringcasecmp-vs-stringdowncase---code)